packages/widget/src/controllers/transfers/fungible-token-transfer.ts

```diff
@@ -105,8 +105,10 @@ export class FungibleTokenTransferController implements ReactiveController {
     this.host.requestUpdate();
   }
 
-  reset(): void {
-    this.sourceNetwork = undefined;
+  reset({ omitSourceNetworkReset } = { omitSourceNetworkReset: false }): void {
```

Comment:
if this is false by default, wouldn't make sense to define it as optional parameter and type cast the undefined? 🤔